test_util/test_runner.py

```diff
@@ -64,55 +56,29 @@ def integration_test(
         for key, value in add_env.items():
             extra_env = key + '=' + value
             test_env.append(extra_env)
-    test_env = ['export '+e+'\n' for e in test_env]
-    test_env = ''.join(test_env)
-    test_cmd = 'py.test -vv ' + ci_flags + ' /integration_test.py'
-
-    log.info('Building py.test image')
-    # Make a clean docker context
-    temp_dir = tempfile.mkdtemp()
-    cmd_script = """
-#!/bin/bash
-set -euo pipefail; set -x
-{test_env}
-{test_cmd}
-""".format(test_env=test_env, test_cmd=test_cmd)
-    write_string(join(temp_dir, 'test_wrapper.sh'), cmd_script)
-    check_call(['cp', test_script, join(temp_dir, 'integration_test.py')])
-    check_call(['cp', pytest_docker, join(temp_dir, 'Dockerfile')])
-    check_call(['cp', test_server, join(temp_dir, 'test_server.py')])
-    check_call(['cp', test_server_docker, join(temp_dir, 'test_server_Dockerfile')])
-    check_call(['docker', 'build', '-t', 'py.test', temp_dir])
 
-    log.info('Exporting py.test image')
-    pytest_image_tar = 'DCOS_integration_test.tar'
-    check_call(['docker', 'save', '-o', join(temp_dir, pytest_image_tar), 'py.test'])
-
-    log.info('Transferring py.test image')
-    tunnel.write_to_remote(join(temp_dir, pytest_image_tar), join(test_dir, pytest_image_tar))
-    log.info('Loading py.test image on remote host')
-    tunnel.remote_cmd(['docker', 'load', '-i', join(test_dir, pytest_image_tar)])
+    test_wrapper = """#!/bin/bash
+{env}
+cd /opt/mesosphere/active/dcos-integration-test
+py.test -vv {ci_flags}
+"""
+    test_env_str = ''.join(['export '+e+'\n' for e in test_env])
+    write_string('test_wrapper.sh', test_wrapper.format(env=test_env_str, ci_flags=ci_flags))
```

Comment:
It would be cool if we had a script in /opt/mesosphere/bin that did exactly this, so we could SSH to a box in the cluster and just run the test. Can we include the test env as part of the package? Or as part of a separate package this is this shell script?